OMCompiler/Compiler/Util/Util.mo

```diff
@@ -919,6 +919,7 @@ algorithm
   xmlString := System.stringReplace(xmlString, "\"", "&quot;");
   xmlString := System.stringReplace(xmlString, "<", "&lt;");
   xmlString := System.stringReplace(xmlString, ">", "&gt;");
+  xmlString := System.stringReplace(xmlString, "\n", "&#10;");
```

Comment:
Are other line endings an issue here? `\r\n` in particular.

Reply:
That is a good question. I am trying to wrap my head around this. The generated files all have `\n` as line endings. We do not add `\r` anywhere. However, we will definitely parse strings (in Modelica files) with `\r` on Windows at some point. So you are right we should handle those as well. To be consistent if nothing else.